desktop/release-notes.md

```diff
@@ -0,0 +1,782 @@
+---
+description: Release notes for Docker Desktop for Mac, Linux, Windows
+keywords: Docker desktop, release notes, linux, mac, windows
+title: Docker Desktop release notes
+toc_max: 2
+redirect_from:
+- /docker-for-mac/release-notes/
+- /docker-for-mac/edge-release-notes/
+- /desktop/mac/release-notes/
+- /docker-for-windows/edge-release-notes/
+- /docker-for-windows/release-notes/
+- /desktop/windows/release-notes/
+- /desktop/linux/release-notes/
+---
+
+This page contains information about the new features, improvements, known issues, and bug fixes in Docker Desktop releases.
+
+> **Note**
+>
+> The information below is applicable to all platforms, unless stated otherwise.
+
+Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/projects/1){: target="_blank" rel="noopener" class="_"} to see what's coming next.
+
+## Docker Desktop 4.8.2
+2022-05-17
+
+> Download Docker Desktop 
+>
+> [For
+> Windows](https://desktop.docker.com/win/main/amd64/Docker%20Desktop%20Installer.exe?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-win-amd64){:
+> .button .primary-btn }
+> [Mac with Intel chip](https://desktop.docker.com/mac/main/amd64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-amd64){: .button .primary-btn }
+> [Mac with Apple
+> chip](https://desktop.docker.com/mac/main/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64){:
+> .button .primary-btn }
+> [DEB](https://desktop.docker.com/linux/main/amd64/docker-desktop-4.8.2-amd64.deb?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-linux-amd64){:
+> .button .primary-btn }
+> [RPM](https://desktop.docker.com/linux/main/amd64/docker-desktop-4.8.2-x86_64.rpm?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-linux-amd64){:
+> .button .primary-btn }
+> [Arch package](https://desktop.docker.com/linux/main/amd64/docker-desktop-4.8.2-x86_64.pkg.tar.zst?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-linux-amd64){:
+> .button .primary-btn }
+
+### Bug fixes and minor changes
+
+- Fixed an issue with manual proxy settings which caused problems when pulling images. Fixes [docker/for-win#12714](https://github.com/docker/for-win/issues/12714) and [docker/for-mac#6315](https://github.com/docker/for-mac/issues/6315).
+- Fixed high CPU usage when extensions are disabled. Fixes [docker/for-mac#6310](https://github.com/docker/for-mac/issues/6310).
+- Docker Desktop now redacts HTTP proxy passwords in log files and diagnostics.
```

Comment:
There is no «For all platforms» here, is it intentional because there is no other «For …» section?

Reply:
Yes intentional. There is a note at the top of the page that says 'The information below is applicable to all platforms, unless stated otherwise.'